wwwroot/sign-in.txt

```diff
@@ -1,6 +1,12 @@
+<<<<<<< HEAD
+       NAME        :   GitHub USERNAME 
+  Brian Bansenauer : Bansenauer-Cascadia
+  Parker Williams  : ParkerW98
+=======
```

Comment:
Parker - you need to edit the sign-in.txt file before you commit it to be sure that your final code does NOT include any of the `<<<<<<< HEAD` or `=======` or `>>>>>>> ef4f233084cfc4873c9a20679d73a9deb5013a20` merge marker text.

Integrate your changes and my changes into one complete list, rather than two separate ones